ios/Exponent/ExponentView/ExponentViewManager.m

```diff
@@ -115,6 +116,12 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
   [[EXRemoteNotificationManager sharedInstance] handleRemoteNotification:notification fromBackground:isFromBackground];
```

Comment:
Done. Although I am not sure what's the `if (remoteNotification || badgeNumber > 0)` check responsible for (the 2nd part of it). Seems that in case of `nil || 1` it will pass null remote notification to the client. Is that desirable?